seed/serialize.py

```diff
@@ -133,19 +138,111 @@ def serialize_and_save_variations_seed_message(seed_data, path):
         if 'max_os_version' in study_data['filter']:
             study.filter.max_os_version = study_data['filter']['max_os_version']
 
-    # Serialize and save
-    with open(path, "wb") as file:
-        file.write(seed.SerializeToString())
-        file.close()
+    return seed
+
+
+def makeFieldTrialTestingConfig(seed, version_string, channel_string):
+    target_version = version.parse(version_string)
+    target_channel = SUPPORTED_CHANNELS[channel_string]
+    config = {}
+    exp_count = 0
+    for study in seed.study:
+        json_study = {}
+        if study.filter.min_version and target_version < version.parse(study.filter.min_version):
+            print('skip ' + study.name + ' because of min_version')
+            continue
+        if study.filter.max_version and target_version > version.parse(study.filter.max_version):
+            print('skip ' + study.name + ' because of max_version')
+            continue
+        if len(study.filter.channel) != 0 and \
+                not target_channel in study.filter.channel:
+            print('skip ' + study.name + ' because of channel')
+            continue
+
+        if len(study.filter.platform) != 0:
+            platforms_json = []
+            platform_names = {
+                study_pb2.Study.Platform.PLATFORM_WINDOWS: 'windows',
+                study_pb2.Study.Platform.PLATFORM_MAC: 'mac',
+                study_pb2.Study.Platform.PLATFORM_LINUX: 'linux',
+                study_pb2.Study.Platform.PLATFORM_IOS: 'ios',
+                study_pb2.Study.Platform.PLATFORM_ANDROID: 'android'
+            }
+            for platform in study.filter.platform:
+                platforms_json.append(platform_names[platform])
+            json_study['platforms'] = platforms_json
+
+        experiments_json = {}
+        best_experiment = max(
+            study.experiment, key=lambda x: x.probability_weight)
+
+        exp_count += 1
+        experiments_json['name'] = 'e' + str(exp_count)
```

Comment:
Names `e<number>` is used.
1. the original names contains weird things and chromium scripts can pass them via cmd
2. It reduces cmd size (currently about ~3K symbols, 8K os limit for win)

Original names are saved as `full_name`
Example of a bad name:
![image](https://user-images.githubusercontent.com/45488748/197849114-3608865d-7bc7-4b50-9ca2-8b528b737223.png)